scripts/tests/direct-download-roaming-network.sh

```diff
@@ -17,7 +17,7 @@ docker network disconnect firezone_app firezone-client-1 # Disconnect the client
 sleep 1
 
 docker network connect firezone_app firezone-client-1 --ip 172.28.0.200 # Reconnect client with a different IP
-sudo kill -s HUP "$(ps -C firezone-linux-client -o pid=)" # Send SIGHUP, triggering `reconnect` internally
+sudo kill -s HUP "$(ps -C firezone-headless-client -o pid=)" # Send SIGHUP, triggering `reconnect` internally
```

Comment:
It's failing on this line, I guess because the old images still call it `firezone-linux-client`

Reply:
To fix a similar issue with the relay, I've learned that there is the `docker compose kill --signal` command. That should work independently of the binary name in the container. Have a look at the graceful-shutdown integration test!